### PR TITLE
docs: team ora documentation

### PR DESCRIPTION
**Pull request by jansenk**
Documentation for setting up, configuring, and managing team ORAs

### CODE REVIEW

File: en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst [hunk 1]
```diff
@@ -214,6 +228,34 @@ for the assignment. These settings include the type of response that learners
...
+Students can be in one team per team-set, so configuring multiple team-sets
+allows you to create groups for different assignments. You could, for example,
+create separate team-sets for Homework, Midterms, and the Final Project,
+each selected for the relevant Team ORAs to allow students to collaborate with
+different classmates for each assignment type. For more on how to configure
+team-sets for your course, see :ref:`Enable and Configure Teams`\.
+
+For Team ORAs, File Upload is the main intended form of response. Learners on a
+team can collaboratively upload and review files as part of a team response.
+A text response is still acceptable but can only be drafted by one member of the
```
Comment (nsprenkle) [inline en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst]: Wording option for clarity?
```suggestion
A text response is still acceptable but can only be drafted by the submitting member of the
```

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 2]
```diff
@@ -133,6 +133,25 @@ To perform a staff assessment in an assignment, follow these steps.
...
+Team-based open response assessments can only be assessed by staff.
+Learners will see this as a Staff Assessment step in their assignment.
+Course team members will see a Grade Available Responses option at the end of the
+assignment.
+
+..note:: Staff assessments for Team ORAs follow the same process as for individual
+  ORAs except that the same score will be applied to all members of the submitting team.
+
+To perform a staff assessment in a team assignment, follow the steps outlined in
+Perform a Staff Assessment. Before submitting, you will receive an alert to notify
```
Comment (nsprenkle) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: Nit: link or bold "Perform a Staff Assessment" to make it clear this is a section in the doc.

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 3]
```diff
@@ -210,6 +229,56 @@ To perform an override assessment, follow these steps.
    final grade on the assessment is updated to reflect the most recent staff
    override assessment grade.
 
+.. Overriding a Team's Assessment Grade:
+
+**************************************
+Overriding a Team's Assessment Grade
+**************************************
+
+As with individual ORAs, you can override the final grade for a team ORA if it
```
Comment (nsprenkle) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: Nit: consistent capitalization. Same on lines 245, 253, and 279
```suggestion
As with individual ORAs, you can override the final grade for a Team ORA if it
```

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 4]
```diff
@@ -299,6 +368,69 @@ was cancelled, and the comments relating to the removal of their submission.
     removal of his submission.
    :width: 500
 
+.. _Remove a Team's Response:
+
+**************************
+Remove a Team’s Response
+**************************
+
+Removing a Team's response is very similar to removing a single learner's response.
```
Comment (nsprenkle) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: Nit: capitalization?
```suggestion
Removing a team's response is very similar to removing a single learner's response.
```

File: en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst [hunk 5]
```diff
@@ -299,6 +368,69 @@ was cancelled, and the comments relating to the removal of their submission.
...
+To remove a submitted Team ORA response, follow these steps.
+
+#. Identify a learner on the team that needs their submission removed from the staff
+   grading workflow. If you only know the team name, use the Teams browser to find
+   learners on a specific team. You can pick the username of any learner on the team
+   by hovering over the learner avatar icons in the Team Details panel.
```
Comment (nsprenkle) [inline en_us/shared/subsections/open_response_assessments/Manage_ORA_Assignment.rst]: Bold sections that are named:

```suggestion
#. Identify a learner on the team that needs their submission removed from the staff
   grading workflow. If you only know the team name, use the **Teams** browser to find
   learners on a specific team. You can pick the username of any learner on the team
   by hovering over the learner avatar icons in the **Team Details** panel.
```

**Review by nsprenkle — APPROVED** [review]
Looks good with minor notes on formatting consistency, specifically:
- Consistently **bolding** named sections & buttons
- Consistent capitalizations for Team vs team. Looks like we use capital Team for "Team ORAs/open response assessments" and named buttons/dialogues where Team is capitalized. Otherwise, lower cased. As long as it's consistent.

File: en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst (hunk 1 above)
Comment (jansenk) [reply en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst]: how would you feel about meeting in the middle with "The one submitting"?

**Pull request merged**